### PR TITLE
Updated the COBS to include Github Actions

### PR DESCRIPTION
**Pull request by prateekshadas**
Github Actions for CI Multiplatform Build 
Ignore Circleci Build Error -> It tries to build the project even though there is no config.yml file for it .

---
This change is [Reviewable](https://reviewable.io/reviews/caffeinetv/obs-studio/80)

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -41,7 +42,7 @@ jobs:
       - name: 'Install prerequisite: Pre-built dependencies'
         shell: bash
         run: |
-          curl -L -O https://github.com/obsproject/obs-deps/releases/download/${{ env.MACOS_DEPS_VERSION }}/osx-deps-${{ env.MACOS_DEPS_VERSION }}.tar.gz
+          curl -L -O  https://github.com/obsproject/obs-deps/releases/download/${{ env.MACOS_DEPS_VERSION }}/osx-deps-${{ env.MACOS_DEPS_VERSION }}.tar.gz
```
Comment (Xaymar) [inline .github/workflows/main.yml]: Extra space.

**Review by Xaymar — APPROVED** [review]
LGTM. Should be easy to rebase onto future versions as well in case OBS releases a new version.

File: .github/workflows/main.yml (hunk 1 above)
Comment (prateekshadas) [reply .github/workflows/main.yml]: Removed extra space

**Review by Xaymar — APPROVED** [review]
LGTM

**Review by PauKerr — APPROVED** [review]

**Pull request merged**